eberron.md

```diff
@@ -84,7 +84,16 @@ For each level cast above cantrip, the range increases by 2 tiles.
 - **Range:** 10 miles
 - **Duration:** 1 hour
 
-You can feel the life force of a creature with whom you have acquired their blood. Target a creature with whom you have their blood. This spell automatically succeeds. You know the general direction of the creature within the range of the spell. You will feel nothing if the creature is dead or outside the range of the spell.
+You can feel the life force of a creature with whom you have acquired their blood. Target a creature with whom you have their blood. Make a spellcasting roll against a bad die. You gain an idea of their location based on the number of successes. You will feel nothing if the creature is dead or outside the range of the spell.
```

Comment:
Up to you what you want here. I think the reasoning for always succeeding is that the difficulty of the spell comes from getting the blood in the first place? If thats the case you can just remove the challenge dice.

I like the accuracy based on success as it rewards invested casters. In Farhome, anyone that has the spell prepared (whether they're trained or not) can cast spells, so I usually codify my spells in a way to reward high successes.